### PR TITLE
docs: Fix upgrade-validator.md regard to migrating genesis

### PR DESCRIPTION
**Pull request by youngjoon-lee**
This issue has been discussed:
- https://github.com/medibloc/panacea-core/issues/154#issuecomment-875399265
- https://github.com/medibloc/panacea-core/issues/154#issuecomment-875428466

### CODE REVIEW

File: panacea-3/upgrade-fullnode.md [hunk 1]
```diff
@@ -30,11 +30,17 @@ make install
 panacead version --long
 ```
 
-5. Rename the config directory and reset state.
+5. Create a new `~/.panacea` directory and copy previous config files.
 ```bash
-mv ~/.panacead ~/.panacea
+MONIKER=$(grep "^moniker = " ~/panacea-2-backup/.panacead/config/config.toml | awk '{print $3}' | sed 's|"||g')
```
Comment (gyuguen) [inline panacea-3/upgrade-fullnode.md:46]: `3. Back up your `~/.panacead` directory`
How about providing the backup creation folder name in bash below here?

```bash
mkdir -p ~/panacea-2-backup
cp -r ~/.panacead ~/panacea-2-backup
cp -r ~/.panaceacli ~/panacea-2-backup
```

Comment (youngjoon-lee) [reply panacea-3/upgrade-fullnode.md:46]: Yeah. I will enhance the explanation.

**Review by gyuguen — APPROVED** [review]
👍 
Thank you.

File: panacea-3/upgrade-fullnode.md (hunk 1 above)
Comment (youngjoon-lee) [reply panacea-3/upgrade-fullnode.md:46]: Done bd76cea. I made it much simpler and clearer.

**Pull request merged**